internal/template/template_utils.go

```diff
@@ -116,3 +117,17 @@ func escapeCharactersForJson(rawString string) (string, error) {
 func escapeNewlines(rawString string) string {
 	return strings.ReplaceAll(rawString, "\n", `\n`)
 }
+
+// EscapeJinjaTemplates replaces each occurrence of "{{" with "\{\{" and each occurrence of "}}" with "\}\}"
+func EscapeJinjaTemplates(src []byte) []byte {
+	src = bytes.ReplaceAll(src, []byte(`{{`), []byte(`\{\{`))
```

Comment:
nit: using a raw string here and regular strings and escaped escape chars for all other cases is a bit strange, any reason?

Reply:
I meant that this one is a back-tick raw string, while the ones on lines 124, 130, 131 are regular strings

```go
src = bytes.ReplaceAll(src, []byte(`{{`), []byte(`\{\{`))
src = bytes.ReplaceAll(src, []byte("}}"), []byte("\\}\\}"))
[...]
str = bytes.ReplaceAll(str, []byte("\\{\\{"), []byte("{{"))
str = bytes.ReplaceAll(str, []byte("\\}\\}"), []byte("}}"))
```

you could probably use raw strings everywhere and skip the escaped-escape :) 


```go
src = bytes.ReplaceAll(src, []byte(`{{`), []byte(`\{\{`))
src = bytes.ReplaceAll(src, []byte(`}}`), []byte(`\}\}`))
[...]
str = bytes.ReplaceAll(str, []byte(`\{\{`), []byte(`{{`))
str = bytes.ReplaceAll(str, []byte(`\}\}`), []byte(`}}`))
```